### DISCUSSION TITLE
Source of national accounts data

### DISCUSSION THREAD

**Discussion by jdebacker** [EAPD-DRB/OG-PHL#18] [Q&A]
What's the best source to find national account data for Philippines?  

Ideally this would be broken out by production sector and include measures of value added and employee compensation.

**Comment by ptuano** [comment]
Thanks @jdebacker ! Of course, the data can be found at the [national accounts page ](https://psa.gov.ph/statistics/national-accounts) of the Philippine Statistics Authority.[ ]

**Reply by marcelolafleur** [reply]
@ptuano Thanks for this link. I see the VA shares by three sectors in the file `Annual Gross Domestic Product Series (1946 to 2023)`: 
- Agriculture, Forestry and Fishing  
- Industry 
- Services

There are also several sectors reported separately but it is easy enough to compute the shares for these sectors: 
- Accommodation and Food Service Activities
- Agriculture, Forestry, and Fishing
- Education
- Electricity, Steam, Water and Waste Management
- Financial and Insurance Activities
- GFCF and Construction
- Human Health and Social Work Activities
- Information and Communication
- Manufacturing
- Mining and Quarrying
- Other Services
- Real Estate and Ownership of Dwellings
- Transportation and Storage
- Wholesale and Retail Trade; Repair of Motor Vehicles and Motorcycles


I don't see anything for the employee compensation but we have someone from department of labor here. I'll ask.

**Comment by marcelolafleur** [comment]
Wages by industry is available here, but maybe there is a better source/format: [Occupational Wages Survey](https://openstat.psa.gov.ph/PXWeb/pxweb/en/DB/DB__1B__OWS/?tablelist=true&_gl=1*uye84u*_ga*MjExNDc1Njk3Mi4xNzI5NDgxMjc1*_ga_EGEWF45N3M*MTcyOTQ4MTI3NC4xLjEuMTcyOTQ4MjI4MS40LjAuMA..)

**Comment by ptuano** [comment]
Here's the PSA webpage on the latest iteration of the Philippines [input-output tables](https://psa.gov.ph/statistics/supply-and-use-input-output). The latest year though is 2018.

**Reply by marcelolafleur** [reply]
Thanks Randy. FYI, OG-PHY is using the 2018 IFPRI-prepared SAM, available here: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/XSEXEP
